lib/plugins/provisioning/src/index.js

```diff
@@ -80,6 +82,11 @@ const storeAllDefaultPlans = (cb = () => {}) => {
     storeAllPlansOfType(type, countCb);
 };
 
+const assureAuthorized = (req, scope) => {
```

Comment:
`assure` prefix implies that we check if the request was authorized. Instead we only authorize. We might consider renaming this to `authorize`